pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java

```diff
@@ -106,67 +106,82 @@ public PulsarClientTool(Properties properties) {
         this.tlsTrustStorePath = properties.getProperty("tlsTrustStorePath");
         this.tlsTrustStorePassword = properties.getProperty("tlsTrustStorePassword");
 
+        initJCommander();
+    }
+
+    protected void initJCommander() {
```

Comment:
not sure. It's a init method, not a reset one because the final state (after the method execution) is ready and not empty (after a reset method I expect a non initialized state)